### ISSUE TITLE
Accessibility test failed

### ISSUE THREAD

**Issue by FatimaDemina** [Gotfrid/flumq-landing-page#36]
<img width="1440" alt="Screen Shot 2022-12-18 at 9 21 02 AM" src="https://user-images.githubusercontent.com/112602396/208272684-9740af57-2a27-4779-bb9f-8a4e68502af8.png">

Please check one accessibility test that we failed in Safari.
